WooCommerce/Classes/ViewRelated/Dashboard/Settings/In-Person Payments/InPersonPaymentsLearnMore.swift

```diff
@@ -36,24 +30,10 @@ private enum Localization {
         comment: "Generic error message when In-Person Payments is unavailable"
     )
 
-    static let learnMore: NSAttributedString = {
-        let learnMoreText = NSLocalizedString(
-            "<a href=\"https://woocommerce.com/payments\">Learn more</a> about accepting payments with your mobile device and ordering card readers",
-            comment: "A label prompting users to learn more about card readers with an embedded hyperlink"
-        )
-
-        let learnMoreAttributes: [NSAttributedString.Key: Any] = [
-            .font: StyleManager.footerLabelFont,
-            .foregroundColor: UIColor.textSubtle
-        ]
-
-        let learnMoreAttrText = NSMutableAttributedString()
-        learnMoreAttrText.append(learnMoreText.htmlToAttributedString)
-        let range = NSRange(location: 0, length: learnMoreAttrText.length)
-        learnMoreAttrText.addAttributes(learnMoreAttributes, range: range)
-
-        return learnMoreAttrText
-    }()
+    static let learnMore = NSLocalizedString(
+        "TEMPORARY CHANGE TO AVOID CRASH - DO NOT MERGE.",
```

Comment:
@koke - I had to make this change to avoid the attribute string crash. I've opened https://github.com/woocommerce/woocommerce-ios/issues/5031 to fix this.